ui/packages/catalog-ui-search/src/main/webapp/component/layer-item/layer-item.collection.view.js

```diff
@@ -14,12 +14,24 @@
  **/
 /* global require*/
 const _ = require('underscore')
+const React = require('react')
```

Comment:
❓ why not `import React from 'react'`?

Reply:
Why would I use one over the other?

Reply:
my thought was consistency across react migrations

Reply:
`import` is an ES6 standard where as `require` is a library. I would suggest going with the standard.